pkg/tsoutil/tso.go

```diff
@@ -44,6 +44,11 @@ func GenerateTS(ts *pdpb.Timestamp) uint64 {
 	return uint64(ts.GetPhysical())<<18 | uint64(ts.GetLogical())&0x3FFFF
 }
 
+// ComposeTS generate an `uint64` TS by passing the physical and logical parts.
+func ComposeTS(physical, logical int64) uint64 {
+	return uint64(physical)<<18 | uint64(logical)&0x3FFFF
```

Comment:
How about call `ComposeTS(ts.GetPhysical(), ts.GetLogical())` in the previous function `GenerateTS` , since the logical are quite similar between L49 and L44